README.md

```diff
@@ -160,13 +110,15 @@ Just add a Froala type in your form:
 ```php
 use KMS\FroalaEditorBundle\Form\Type\FroalaEditorType;
 
-$builder->add('yourField', FroalaEditorType::class);
+$builder-add( "Froalaeditor", FroalaEditorType::class,[
```

Comment:
I think this is specific to your configuration

Reply:
@jmsche will undo it.